README.md

```diff
@@ -1,2 +1,52 @@
 # douglass
 Jupyter notebook preview service
+
+### Description
+Your web-app needs to display read-only versions of Jupyter notebooks. 
+
+Under the hood, Jupyter .ipynb files are JSON documents with a particular [format](https://nbformat.readthedocs.io). Jupyter provides a commandline tool / library [nbconvert](https://nbconvert.readthedocs.io) for interacting with these documents and converting them to various formats, in particular HTML.
+
+This project is essentially nbconvert as a REST service. It takes notebook JSON and returns HTML that can be embedded in your application.
+
+There are two routes:
+
+/status is a health-check endpoint. It accepts GET requests and returns 200 'OK'.
+
+/api/convert is the primary conversion endpoint. It accepts POST requests containing notebook JSON and returns HTML for displaying the notebook's content in the browser.
+This endpoint is authorized via Terra's SAM authorization service. Calls therefore require an auth header supplying a Terra Oauth2 bearer token.
```

Comment:
does it have to be a `Terra` oauth token?

Reply:
I'll clarify the comment. It's an Oauth2 token representing the user's Google identity, either from Google SSO or gcloud auth login. That Google identity must be known to Terra (user must be registered and enabled in SAM)